src/css/memberTeams.css

```
@@ -0,0 +1,116 @@
+.teams-page {
+  margin-top: 75px;
```

We do not need this much marging. Might want to reduce this value because it looks like too much spacing.

---

Set this to 12px. Similar to the MemberInfoContainer `.member-info-container`
```suggestion
  margin-top: 12px;
```